video_xblock/static/vendor/js/Vimeo.js

```diff
@@ -390,8 +397,8 @@ THE SOFTWARE. */
               '.vjs-vimeo .vjs-iframe-blocker { display: none; }' +
               '.vjs-vimeo.vjs-user-inactive .vjs-iframe-blocker { display: block; }' +
               '.vjs-vimeo .vjs-poster { background-size: cover; }' +
-              // '.vjs-vimeo { height:100%; }' +
-              '.vimeoplayer { width:100%; height:180%; position:absolute; left:0; top:-40%; }';
+              '.vimeoplayer { width:100%; height:100%; position:absolute; left:0; top:0; pointer-events: none;}' +
+              '.video-js .vjs-control-bar { z-index: 3 }';
```

Comment:
z-index just a three. Do you think this is a robust way? Does it make sense to increase the value? As for me I usually set custom z-index pretty big to be sure that my custom always be on a top 🙂

Reply:
I decided that in this case, it's sufficient value.